github-actions-utils/healthcheck.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+nodeReady=1
+echo "Waiting for node to finish initializing..."
+until [ $nodeReady -eq 0 ]
+do
+  grep 'Now current platform status = ACTIVE' ../compose-network/node0/output/hgcaa.log
+  nodeReady=$?
+  sleep 3s
```

Comment:
Use this form so the command works on MacOS and Linux:

```suggestion
  sleep 3
```